cfme/utils/appliance/console.py

```diff
@@ -59,7 +59,7 @@ def run_commands(self, commands, autoreturn=True, timeout=10, channel=None, outp
             stdout.append(result)
             logger.debug(result)
         if output:
-            logger.info("commands output: %s" % result)
+            logger.info("commands output: %s" % stdout)
```

Comment:
This should just be result, we don't want to print the entire stout everytime

Reply:
We are interested in CLI behavior for entire commands not only for the last command which we executed from command list.

Reply:
We are just printing `stdout` once after all commands execution is done.

Reply:
@dgaikwad After seeing your explanation, I brought up this up in the reviewers call and we are suggesting the following updates:
1) Line 60, change this from `debug` to `info`. We should be logging the result of each command on each iteration
2) Remove `output` as an arg and the if statement on line 61. Code on line 62 and 62 should always be run.
3) Line 37, we should be creating a dict here instead, the key being the command, and the value being the result. Then log that dict on line 62. 

I hope that makes sense!

Reply:
@jawatts Thank you for the first two suggestion but in third suggestion, if we make dict instead of list then here we are going to override result.
Example:
`commands = ("ap", "", "5", "2", "1", "2", "2")
`
In above example we will not able to store result separately for all `2` and `1` command and we will not get which command executed first due to its `dict` type.